### DISCUSSION TITLE
Do we need FFT.cs?

### DISCUSSION THREAD

**Discussion by dsn27** [FriendsOfCADability/CADability#211] [General]
Is the file FFT.cs needed?

I couldn't find any references in the project to this class/file.

This is not CLS-compliant (CS3001)
https://github.com/FriendsOfCADability/CADability/blob/728d71d716b169bbf6261d31860d48d9373354ca/CADability/FFT.cs#L59-L66

**Comment by stefan-tb** [comment]
Probably not. By all means running an FFT is expensive and rather done in highly optimized libs (see IPP, Intel Performance Primitives) instead of C#.

**Comment by dsn27** [comment]
I removed the class
